mteb/tasks/Clustering/dan/WikiClusteringP2P.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class WikiClusteringP2P(AbsTaskClustering):
+    metadata = TaskMetadata(
+        name="WikiClusteringP2P",
+        description="Clustering of wikipedia articles inspired by BlubrbsClusteringP2P. Labels are taken from top-level categories as defined in https://da.wikipedia.org/wiki/Kategori:Topniveau_for_emner.",
+        reference=None,
+        dataset={
+            "path": "ryzzlestrizzle/dawiki-clustering-p2p",
+            "revision": "4c4c6b6ad63311d6a072734d86dca1c0b48c441c",
+        },
+        type="Clustering",
+        category="p2p",
+        eval_splits=["test"],
+        eval_langs=["dan-Latn"],
```

Comment:
```suggestion
        eval_langs={"{lang_name_on_hf}": "dan-Latn", ...},
```
When you transform it into a multilingual dataset, the list should be converted to a mapping.

Reply:
Great idea! I'll update the pipeline to accommodate:))